Subject: input: do not include input-compat.h
From: Jiri Slaby <jslaby@suse.cz>
Patch-mainline: never

Including the file causes kabi changes:
modversion changed because of changes in struct compat_timespec (became defined), struct compat_robust_list_head (became defined).

Introduced by e1155f2ae2c0562b4efe8b2b5165ad911be0d008.

Signed-off-by: Jiri Slaby <jslaby@suse.cz>
---
 drivers/input/input.c |    2 ++
 1 file changed, 2 insertions(+)

--- a/drivers/input/input.c
+++ b/drivers/input/input.c
@@ -24,7 +24,9 @@
 #include <linux/mutex.h>
 #include <linux/rcupdate.h>
 #include <linux/smp_lock.h>
+#ifndef __GENKSYMS__
 #include "input-compat.h"
+#endif
 
 MODULE_AUTHOR("Vojtech Pavlik <vojtech@suse.cz>");
 MODULE_DESCRIPTION("Input core");
